### PR TITLE
[MI-3321] Post a DM message on the successful connection of the user with MS Teams

### PR DESCRIPTION
**Pull request by ayusht2810**
#### Summary
Post a DM message on the successful connection of the user with MS Teams.
image

### CODE REVIEW

File: server/api.go [hunk 1]
```diff
@@ -412,6 +412,8 @@ func (a *API) oauthRedirectHandler(w http.ResponseWriter, r *http.Request) {
 		&model.WebsocketBroadcast{UserId: mmUserID},
 	)
 
+	_, _ = a.p.DM(mmUserID, "You've successfully connected your Mattermost account to MS Teams.")
```
Comment (raghavaggarwal2308) [inline server/api.go:415]: Check error

**Review by raghavaggarwal2308 — CHANGES_REQUESTED** [review]

File: server/bot.go [hunk 2]
```diff
@@ -0,0 +1,28 @@
+package main
+
+import (
+	"fmt"
+
+	"github.com/mattermost/mattermost-server/v6/model"
+)
+
+// DM posts a simple Direct Message to the specified user
```
[the remark this answers is not in the document]
Comment (ayusht2810) [reply server/bot.go:9]: I think it is already fine like this

**Review by raghavaggarwal2308 — APPROVED** [review]

**Review by manojmalik20 — APPROVED** [review]

**Comment by lieut-data** [comment]
@ayusht2810 / @johndavidlugtu, can you confirm this is still a product requirement? I'm guessing it comes out of a flow where the user connects not via the slash command but via the RHS or some other UI?

**Comment by ayusht2810** [comment]
@lieut-data I think we can close this and other UI related PRs now, as the [figma designs](https://www.figma.com/file/aRgXgPZQ3lDaow5uatDzz9/MS-Teams-Sync-RHS?type=design&node-id=1-7&mode=design&t=ewtCZk3sFs0sMBLl-0) for the RHS is updated now.

**Comment by lieut-data** [comment]
Thanks, @ayusht2810! Please do feel free to close any PRs that are no longer applicable especially if requirements have changed.

**Comment by ayusht2810** [comment]
closing this PR in favour of updated [figma designs](https://www.figma.com/file/aRgXgPZQ3lDaow5uatDzz9/MS-Teams-Sync-RHS?type=design&node-id=1-7&mode=design&t=ewtCZk3sFs0sMBLl-0)

**Pull request closed without merging**